onyo/commands/unset.py

```diff
@@ -20,7 +20,7 @@
         nargs='+',
         type=str,
         help="""
-            KEYs to unset in assets. Multiple keys can be given
+            Keys to unset in assets. Multiple keys can be given
             (e.g. key1 key2 key3).
```

Comment:
For onyo set the example for multiple keys was ticks-quoted in this stile: "(e.g. ``key1 key2 key3``)."

Reply:
Fixed and pushed.